### DISCUSSION TITLE
请问目前对【阅读】的书源适配情况如何

### DISCUSSION THREAD

**Discussion by 782669218** [aooiuu/any-reader#152] [Q&A]
### 功能描述

非常感谢开发这样一个开源阅读软件，并且还支持了vs插件，但是我发现目前好像没有一个可用的书源。看以前的Issue看到有人提过对【阅读】书源的适配，也看到Issue中说明适配的难度比较大，但是我觉得目前这类软件太依赖社区有大佬贡献源了，一个新软件如果不适配已存在的源的话，真的太难发展了。

**Comment by aooiuu** [comment]
阅读规则的解析，目前仅实现了部分规则的搜索。代码在` packages/legado`，感谢  Horis 大佬贡献的代码。欢迎大佬们继续完善。

亦搜规则生态确实不如阅读，但好在他全平台。亦搜的规则模型也能处理大部分场景。

any-reader 的长久目标是支持多种规则的解析，不仅限于亦搜。

阅读的代码我还不够了解，看有没有有缘人继续完善。何时能适配，目前还不好说，个人精力有限。
